helpers/mathjax.js

```diff
@@ -32,9 +32,11 @@ export class HtmlBlockMathRenderer {
 		await loadMathJax(mathJaxConfig);
 
 		const temp = document.createElement('div');
+		temp.style.display = 'none';
 		temp.attachShadow({ mode: 'open' });
 		temp.shadowRoot.innerHTML = `<div><mjx-doc><mjx-head></mjx-head><mjx-body>${elem.innerHTML}</mjx-body></mjx-doc></div>`;
 
+		elem.appendChild(temp);
```

Comment:
Similar comments RE: the note above. Similarly as well, two lines below we ignore this temp element and return only the child's shadowRoot, which we then use to overwrite `elem` entirely. I think we're safe from any long-term side effects here.